### PR TITLE
FS-853: Create BOM service table scheme and generate ORM files.

### PR DESCRIPTION
**Pull request by Alva8756**
This PR create sqlboiler sql file and generate related ORM model files for BOM service.
As listed in the design doc, we will need to create 3 tables `bom-info`, `aoc_mac_address_bom` and `bmc_mac_address_bom`.

More table details can be found in design doc: [19IG7fU0jc_F3GV71nx_P5hbMHjhCv3_EzIUmFfvCGNU.](url)

Tested in Sandbox: 
   When using the locally built serverService image in sandbox,
   the crdb initialize the new added tables successfully.
```
root@:26257/defaultdb> SHOW TABLES;
  schema_name |         table_name         | type  | owner | estimated_row_count | locality
--------------+----------------------------+-------+-------+---------------------+-----------
  public      | aoc_mac_address            | table | root  |                   0 | NULL
  public      | attributes                 | table | root  |                   0 | NULL
  public      | attributes_firmware_set    | table | root  |                   0 | NULL
  public      | bmc_mac_address            | table | root  |                   0 | NULL
  public      | bom_info                   | table | root  |                   0 | NULL
  public      | component_firmware_set     | table | root  |                   0 | NULL
  public      | component_firmware_set_map | table | root  |                   0 | NULL
  public      | component_firmware_version | table | root  |                   0 | NULL
  public      | goose_db_version           | table | root  |                  22 | NULL
  public      | server_component_types     | table | root  |                   0 | NULL
  public      | server_components          | table | root  |                   0 | NULL
  public      | server_credential_types    | table | root  |                   1 | NULL
  public      | server_credentials         | table | root  |                   0 | NULL
  public      | servers                    | table | root  |                   0 | NULL
  public      | versioned_attributes       | table | root  |                   0 | NULL
(15 rows)
```

### CODE REVIEW

File: db/migrations/00021_bom.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+CREATE TABLE bom_info (
+  serial_num STRING PRIMARY KEY NOT NULL,
+  mac_aoc_address STRING NULL,
+  num_defi_pmi STRING NULL,
+  num_def_pwd STRING NULL,
+  metro STRING NULL
+);
+
+CREATE TABLE mac_aoc_address_bom (
```
Comment (DoctorVin) [inline db/migrations/00021_bom.sql:12]: Minor naming quibbles: consider renaming one of the two Mac tables so they are more consistent? Either `aoc_mac_address` or `mac_bmc_address`? Also, it feels like the `_bom` suffix stutters here (i.e it repeats without adding much value).

File: internal/models/boil_suites_test.go
Comment (DoctorVin) [inline internal/models/boil_suites_test.go:1]: kinda neat that it generates code and tests itself for the tables you created.

**Review by DoctorVin — COMMENTED** [review]
This looks reasonable to me. Have you tried creating these tables and verifying it in the sandbox?

File: db/migrations/00021_bom.sql (hunk 1 above)
Comment (Alva8756) [reply db/migrations/00021_bom.sql]: Done. Thanks!

**Review by DoctorVin — APPROVED** [review]

**Pull request merged**